hcatalog/webhcat/svr/pom.xml

```diff
@@ -104,6 +104,16 @@
     <dependency>
       <groupId>org.apache.zookeeper</groupId>
       <artifactId>zookeeper</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>ch.qos.logback</groupId>
```

Comment:
is this new and required for the 3.9.1?

Reply:
This is required because Zookeeper introduced dependency on logback at [ZOOKEEPER-4427](https://issues.apache.org/jira/browse/ZOOKEEPER-4427) which is not required as [Hive uses log4j2](https://issues.apache.org/jira/browse/HIVE-25795). Excluded logback dependencies similar to that done in [Hadoop](https://github.com/apache/hadoop/pull/6296/files). The logging for Zookeeper client will not get impacted as log4j2 should also be compatible (Ref [here](https://github.com/apache/zookeeper/pull/1793#pullrequestreview-857545860))